infra/keyvault/platform-keyvault.bicep

```diff
@@ -29,14 +29,14 @@ param roleAssignments array
 param skuName string = 'premium'
 
 @description('Required. Date in the format yyyy-MM-dd.')
-param createdDate string = utcNow('yyyy-MM-dd')
+param createdOrUpdatedDate string = utcNow('yyyy-MM-dd')
 
 @description('Required. Date in the format yyyyMMdd-HHmmss.')
 param deploymentDate string = utcNow('yyyyMMdd-HHmmss')
 
 var customTags = {
   Location: location
-  CreatedDate: createdDate
+  CreatedOrUpdatedDate: createdOrUpdatedDate
```

Comment:
sorry can we leave it as createdDate - Ken is technically right, but we should leave alone for now mate, will change later

Reply:
@DaRoszko @kbabigumira Reverted back to CreatedDate.